app/styles/components/_page-layout.scss

```diff
@@ -2,18 +2,25 @@
 
   &.categories-collapsed {
     .content {
-      @include span-columns(12);
+      float: left;
+      display: block;
+      margin-right: 0;
+      width: 100%;
+
       padding: $default-spacing $default-spacing 0 $default-spacing;
     }
   }
 
   .content {
-    @include span-columns(12);
+    float: left;
+    display: block;
+    margin-right: 2.35765%;
```

Comment:
I'd prefer to replace the layout tactic, rather than hardcoding the margin, etc that was generated by `span-columns`